### PR TITLE
enh(release): Prepare Centreon Web 20.04.14 RN

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [x] 20.04.x
- [ ] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x (master)

### CODE REVIEW

**Review by mdebauchez — COMMENTED** [review]
There's a spelling mistake lign 40  in french and english : wizard and not wirard

File: en/releases/centreon-core.md [hunk 1]
```diff
@@ -19,6 +19,37 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Administration] Import of JS in image files
+- [Administration] Insecure media file upload
+- [Administration] SQL Injection on ACL actions
+- [Administration] SQL Injection on ACL resources
+- [Administration] SQL Injection on reload ACL
+- [Configuration] SQL Injection on MediaWiki
+- [Configuration] SQL Injection on SNMP trap manufacturer
+- [Configuration] SQL Injection on poller form
+- [Configuration] Unserialize() are not sanitized in Centreon Broker wizard
+- [Configuration] Unserialize() are not sanitized in poller wirard
```
Comment (thiuyendang) [inline en/releases/centreon-core.md:40]:
```suggestion
- [Configuration] Unserialize() are not sanitized in poller wizard
```

File: fr/releases/centreon-core.md [hunk 2]
```diff
@@ -18,6 +18,37 @@ commerciales, vous pouvez vous rendre sur notre
...
+- [Administration] Import of JS in image files
+- [Administration] Insecure media file upload
+- [Administration] SQL Injection on ACL actions
+- [Administration] SQL Injection on ACL resources
+- [Administration] SQL Injection on reload ACL
+- [Configuration] SQL Injection on MediaWiki
+- [Configuration] SQL Injection on SNMP trap manufacturer
+- [Configuration] SQL Injection on poller form
+- [Configuration] Unserialize() are not sanitized in Centreon Broker wizard
+- [Configuration] Unserialize() are not sanitized in poller wirard
```
Comment (thiuyendang) [inline fr/releases/centreon-core.md:39]:
```suggestion
- [Configuration] Unserialize() are not sanitized in poller wizard
```

File: en/releases/centreon-core.md [hunk 3]
```diff
@@ -19,6 +19,37 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Configuration] XSS reflected on Graph performance curves
+- [Configuration] XSS reflected on SNMP trap
+- [Configuration] XSS reflected on internal API broker configuration
+- [Graph] SQL Injection on Graph component templates
+- [Graph] SQL Injection on Graph generate image
+- [Reporting] SQL Injection on reporting export
+
```
Comment (thiuyendang) [inline en/releases/centreon-core.md:47]:
```suggestion : MON-7199, missing ticket
[SECU][WEB] Packaging, remove . gitignore files
```

File: fr/releases/centreon-core.md [hunk 4]
```diff
@@ -18,6 +18,37 @@ commerciales, vous pouvez vous rendre sur notre
...
+- [Configuration] XSS reflected on Graph performance curves
+- [Configuration] XSS reflected on SNMP trap
+- [Configuration] XSS reflected on internal API broker configuration
+- [Graph] SQL Injection on Graph component templates
+- [Graph] SQL Injection on Graph generate image
+- [Reporting] SQL Injection on reporting export
+
```
Comment (thiuyendang) [inline fr/releases/centreon-core.md:46]:
```suggestion : MON-7199, missing ticket
[SECU][WEB] Packaging, remove . gitignore files
```

```

**Review by thiuyendang — CHANGES_REQUESTED** [review]

File: en/releases/centreon-core.md [hunk 5]
```diff
@@ -19,6 +19,37 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Configuration] SQL Injection on poller form
+- [Configuration] Unserialize() are not sanitized in Centreon Broker wizard
+- [Configuration] Unserialize() are not sanitized in poller wizard
+- [Configuration] XSS reflected on Graph performance curves
+- [Configuration] XSS reflected on SNMP trap
+- [Configuration] XSS reflected on internal API broker configuration
+- [Graph] SQL Injection on Graph component templates
+- [Graph] SQL Injection on Graph generate image
+- [Reporting] SQL Injection on reporting export
+[SECU][WEB] Packaging, remove . gitignore files
```
Comment (tuntoja) [inline en/releases/centreon-core.md:47]:
```suggestion
- [SECU][WEB] Packaging, remove . gitignore files

```

File: fr/releases/centreon-core.md [hunk 6]
```diff
@@ -18,6 +18,37 @@ commerciales, vous pouvez vous rendre sur notre
...
+- [Configuration] SQL Injection on poller form
+- [Configuration] Unserialize() are not sanitized in Centreon Broker wizard
+- [Configuration] Unserialize() are not sanitized in poller wizard
+- [Configuration] XSS reflected on Graph performance curves
+- [Configuration] XSS reflected on SNMP trap
+- [Configuration] XSS reflected on internal API broker configuration
+- [Graph] SQL Injection on Graph component templates
+- [Graph] SQL Injection on Graph generate image
+- [Reporting] SQL Injection on reporting export
+[SECU][WEB] Packaging, remove . gitignore files
```
Comment (tuntoja) [inline fr/releases/centreon-core.md:46]:
```suggestion
- [SECU][WEB] Packaging, remove . gitignore files

```

File: en/releases/centreon-core.md [hunk 7]
```diff
@@ -19,6 +19,37 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 20.04.14
+
+#### Bugfixes
+
+- [Administration] Broker statistics for pollers are not shown
+- [Core] Avoid 404 redirection
+
```
Comment (thiuyendang) [inline en/releases/centreon-core.md:28]: Missing this ticket too:https://centreon.atlassian.net/browse/MON-7368

File: fr/releases/centreon-core.md [hunk 8]
```diff
@@ -18,6 +18,37 @@ commerciales, vous pouvez vous rendre sur notre
 
 ## Centreon Web
 
+### 20.04.14
+
+#### Correctifs
+
+- [Administration] Broker statistics for pollers are not shown
+- [Core] Avoid 404 redirection
+
```
Comment (thiuyendang) [inline fr/releases/centreon-core.md:27]: Missing: https://centreon.atlassian.net/browse/MON-7368

**Review by thiuyendang — CHANGES_REQUESTED** [review]

**Review by thiuyendang — APPROVED** [review]

**Review by tuntoja — APPROVED** [review]

File: fr/releases/centreon-core.md [hunk 9]
```diff
@@ -18,6 +18,42 @@ commerciales, vous pouvez vous rendre sur notre
...
+- [Configuration] SQL Injection on poller form
+- [Configuration] Unserialize() are not sanitized in Centreon Broker wizard
+- [Configuration] Unserialize() are not sanitized in poller wizard
+- [Configuration] XSS reflected on Graph performance curves
+- [Configuration] XSS reflected on SNMP trap
+- [Configuration] XSS reflected on internal API broker configuration
+- [Graph] SQL Injection on Graph component templates
+- [Graph] SQL Injection on Graph generate image
+- [Install] Packaging, remove . gitignore files
+- [Reporting] SQL Injection on reporting export
```
Comment (tuntoja) [inline fr/releases/centreon-core.md:50]:
```suggestion
- [Reporting] SQL Injection on reporting export

```

**Review by tuntoja — CHANGES_REQUESTED** [review]

**Review by tuntoja — APPROVED** [review]

**Pull request merged**